### PR TITLE
docs(presets): add `edit_url` in meta of preset pages

### LINKED ISSUE #25236: Link preset docs to source
### Describe the proposed change(s).

Example preset docs page: https://docs.renovatebot.com/presets-group/

Currently links to https://github.com/renovatebot/renovate/edit/main/docs/usage/presets-group.md

Should link to https://github.com/renovatebot/renovate/edit/main/lib/config/presets/internal/group.ts

### PR DESCRIPTION
**Pull request by RahulGautamSingh**
## Changes
- Add edit_url in meta section of the internal presets pages

## Context
Closes: https://github.com/renovatebot/renovate/issues/25236

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [x] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository
- Confirmed by visting all the links generated

### CODE REVIEW

File: tools/docs/presets.ts [hunk 1]
```diff
@@ -1,23 +1,54 @@
...
+  // switch (newName) {
+  //   case 'regexmanagers':
+  //     return `${url}regex-managers.ts`;
+  //   case 'mergeconfidence':
+  //     return `${url}merge-confidence.ts`;
+  //   case 'full config':
+  //     return `${url}config.ts`;
+  //   default:
+  //     return `${url}${newName}.ts`;
+  // }
```
Comment (RahulGautamSingh) [inline tools/docs/presets.ts:31]: I wasn't sure which approach to use to handle the preset names like `regexManagers` and `mergeConfidence`.

They need to be formatted from `regexManagers` to `regex-managers`. 
We can use regex matching or hard-code using switch statement.

Which do you guys prefer?

Comment (viceice) [reply tools/docs/presets.ts:31]: I think I prefer switch case

**Comment by rarkins** [comment]
How can this be tested? Or for example @RahulGautamSingh can you provide a diff between the old generated file and the new one?

**Comment by RahulGautamSingh** [comment]
> How can this be tested? Or for example @RahulGautamSingh can you provide a diff between the old generated file and the new one?

Did it in reverse ie, generated the new docs files then reverted my changes and generated again (ie. old docs files) and comapred them.
https://gist.github.com/RahulGautamSingh/a1272f5642b9c2e68c993ddce084af15

**Review by rarkins — APPROVED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**